<?xml version="1.0"?>
<launch>
    <!-- ATIM -->
    <param
        name="camera_config_path"
	    value="$(find atim)/configs/gazebo_cam" />
	<param
        name="gimbal_config_path"
	    value="$(find atim)/configs/gimbal/config.yaml" />
	<param
        name="run_in_sim_mode"
	    value="1" />
	<param
        name="ros_image_topic"
        value="/gazebo_client/gimbal_cam_image" />
	<param
        name="gimbal_state_topic"
        value="/gazebo_client/gimbal/camera_imu" />
	<node
	    pkg="atim"
	    type="atim"
	    name="atim"
	    output="screen" />

    <!-- CONTROLLER -->
	<param
        name="quadrotor"
        value="$(find awesomo)/configs/quadrotor/config.yaml" />
	<param
        name="position_controller"
        value="$(find awesomo)/configs/position_controller/simulation_config.yaml" />
	<param
        name="carrot_controller"
        value="$(find awesomo)/configs/carrot_controller/config.yaml" />
	<node
	    pkg="awesomo"
	    type="awesomo_node"
	    name="awesomo"
        output="screen" />

</launch>
